lambda/main.go

```diff
@@ -53,6 +53,10 @@ func handler(ctx context.Context, event cfn.Event) (physicalResourceID string, d
 		if err != nil {
 			return physicalResourceID, data, err
 		}
+		imageArch, err := getStrPropsDefault(event.ResourceProperties, IMAGE_ARCH, "")
```

Comment:
`imageArch` appears to be unused, the logging statement aside.

Reply:
Good catch!  Updated to reference it in the following NewImageOpts calls.